Automattic/wp-calypso
# Jetpack Cloud | Add Jetpack features comparison section

This PR adds a new section to Jetpack Cloud - `/features/comparison`

Discussion p2: p1HpG7-jrS-p2

#### Proposed Changes

* Create jetpack features comparison section components
* Add section to config files
* Create hooks for comparison data

#### Testing Instructions

- Do any one of these
    - Click on Jetpack Cloud live link below and wait for the redirect to complete, then go to `/features/comparison`
    - or boot up this PR 
        - Run `git fetch && git checkout add/jetpack-ft-comparison-section`
        - Run `yarn start-jetpack-cloud`
        - Goto [http://jetpack.cloud.localhost:3000/features/comparison](http://jetpack.cloud.localhost:3000/features/comparison)
- Confirm that the page exists and is not a 404.
- Confirm that the page is visible to both logged in and logged out users
- Change the language at the bottom using language switcher
- Confirm that it works for non-english languages
- Confirm that the design matches Figma both for desktop and mobile - RAs5UTOlm0w6zYP9MifV0r-fi-1%3A29

#### Pre-merge Checklist





Completes 0-as-1203485587457711/f

https://user-images.githubusercontent.com/18226415/209342717-43b043ba-88fb-4240-a187-f21ce830e9a2.mov

<img width="1712" alt="Screenshot 2022-12-23 at 6 48 14 PM" src="https://user-images.githubusercontent.com/18226415/209342656-b71d1b53-621f-4fb2-aeef-fbdd5165faa0.png">
<img width="1714" alt="Screenshot 2022-12-23 at 6 48 32 PM" src="https://user-images.githubusercontent.com/18226415/209342680-65d82f66-b64a-4949-b40d-34cb6a52720e.png">
<img width="1712" alt="Screenshot 2022-12-23 at 6 48 47 PM" src="https://user-images.githubusercontent.com/18226415/209342682-894792d7-a1d4-470d-a0d4-a357e03a5f11.png">


<img width="420" alt="Screenshot 2022-12-23 at 7 00 42 PM" src="https://user-images.githubusercontent.com/18226415/209344137-ea38cbba-4df3-4072-939c-3677612ae9e6.png">
<img width="422" alt="Screenshot 2022-12-23 at 6 58 50 PM" src="https://user-images.githubusercontent.com/18226415/209343886-fb807fb4-469a-4d5b-b03c-9011d92dd268.png">

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1203449037077882
